homeassistant/components/switch/lutron.py

```diff
@@ -0,0 +1,50 @@
+"""
+Support for Lutron switches.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.lutron/
+"""
+import logging
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_CONTROLLER)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['lutron']
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Lutron switches."""
+    devs = []
+    for (area_name, device) in hass.data[LUTRON_DEVICES]['switch']:
+        dev = LutronSwitch(area_name, device, hass.data[LUTRON_CONTROLLER])
+        devs.append(dev)
+
+    add_entities(devs, True)
+    return True
+
+
+class LutronSwitch(LutronDevice, SwitchDevice):
+    """Representation of a Lutron Switch."""
+
+    def turn_on(self, **kwargs):
+        """Turn the switch on."""
+        self._lutron_device.level = 100
+
+    def turn_off(self, **kwargs):
+        """Turn the switch off."""
+        self._lutron_device.level = 0
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attr = {}
+        attr['Lutron Integration ID'] = self._lutron_device.id
```

Comment:
Keys in state attributes should be lowercase snakecase.

Reply:
This was done to remain consistent with the existing light/lutron.py.  I've updated the new switch code.  I could update and include a fix for the light in this pull request as well.

Reply:
Could up make a separate PR with that correction for the light platform? It will be a breaking change.